crdppf/util/pdf_functions.py

```diff
@@ -6,7 +6,9 @@
 # geometry related librabries
 from shapely.geometry import Point as splPoint, Polygon as splPolygon
 from shapely.geometry import MultiPolygon as splMultiPolygon, LinearRing as splLinearRing
-from geoalchemy import WKTSpatialElement
+from geoalchemy2 import functions, WKTElement, WKBElement, shape
```

Comment:
I do not think that you use `WKBElement` and `shape` in this code file ? If not, please remove from the imports...